README.md

```diff
@@ -1,3 +1,166 @@
-# react-native-voice-processor
+# React Native Voice Processor
 
-A native module for reading audio and passing audio buffers to the React Native Javascript layer.
+[![GitHub release](https://img.shields.io/github/release/Picovoice/react-native-voice-processor.svg)](https://github.com/Picovoice/react-native-voice-processor/releases)
+[![GitHub](https://img.shields.io/github/license/Picovoice/react-native-voice-processor)](https://github.com/Picovoice/react-native-voice-processor/)
+
+[![npm](https://img.shields.io/npm/v/@picovoice/react-native-voice-processor)](https://www.npmjs.com/package/@picovoice/react-native-voice-processor)
+
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+<!-- markdown-link-check-disable -->
+[![Twitter URL](https://img.shields.io/twitter/url?label=%40AiPicovoice&style=social&url=https%3A%2F%2Ftwitter.com%2FAiPicovoice)](https://twitter.com/AiPicovoice)
+<!-- markdown-link-check-enable -->
+[![YouTube Channel Views](https://img.shields.io/youtube/channel/views/UCAdi9sTCXLosG1XeqDwLx7w?label=YouTube&style=social)](https://www.youtube.com/channel/UCAdi9sTCXLosG1XeqDwLx7w)
+
+The React Native Voice Processor is an asynchronous audio capture library designed for real-time audio
+processing on mobile devices. Given some specifications, the library delivers frames of raw audio
+data to the user via listeners.
+
+## Table of Contents
+
+- [React Native Voice Processor](#react-native-voice-processor)
+  - [Table of Contents](#table-of-contents)
+  - [Requirements](#requirements)
+  - [Compatibility](#compatibility)
+  - [Installation](#installation)
+  - [Permissions](#permissions)
+  - [Usage](#usage)
+    - [Capturing with Multiple Listeners](#capturing-with-multiple-listeners)
+  - [Example](#example)
+
+## Requirements
+
+- [Node.js](https://nodejs.org) (16+)
+- [Android SDK](https://developer.android.com/about/versions/12/setup-sdk) (21+)
+- [JDK](https://www.oracle.com/java/technologies/downloads/) (8+)
+- [Xcode](https://developer.apple.com/xcode/) (11+)
+- [CocoaPods](https://cocoapods.org/)
+
+## Compatibility
+
+- React Native 0.62.2+
```

Comment:
I think `0.63.5` works better, not really sure but some one these versions have xcode compatibility, we might as well just say `0.68.7` cause I remember that being the latest version with the fix.